extensions/default/src/DicomWebDataSource/index.js

```diff
@@ -17,6 +17,13 @@ import {
 } from './retrieveStudyMetadata.js';
 import StaticWadoClient from './utils/StaticWadoClient.js';
 
+import axios from "axios";
+
+const nlApi = axios.create({
```

Comment:
@adinhodovic I think we do not need this baseURL and withCredentials. We just use "/api/" to call into the backend.

Reply:
I think it needs credentials (it is used this in nl-pacs already). Configurability is just good for other envs